UPGRADING.md

```diff
@@ -287,21 +283,7 @@ Here are some examples:
 ```diff
   // changing severity
 - bugsnagClient.notify(err, { severity: 'info' })
-+ Bugsnag.notify(err, event => { event.severity = 'info' })
-
-  // adding metadata
```

Comment:
Personally I think that this example is useful (in addition to the example about when there is no section)